opencensus/__init__.py

```diff
@@ -1,20 +1 @@
-# Copyright 2017, OpenCensus Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-try:
-    import pkg_resources
-    pkg_resources.declare_namespace(__name__)
-except ImportError:  # pragma: NO COVER
-    import pkgutil
-    __path__ = pkgutil.extend_path(__path__, __name__)
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
Looking at [`extend_path`](https://github.com/python/cpython/blob/2.7/Lib/pkgutil.py#L480-L552) I think you're supposed to include only this line to avoid import-time effects that change the path, but that comments are fine. But there may be some black magic I'm missing here.

Reply:
I thought about this and decided to remove the copyright comment because:
1. If 3rd party extension has the `__init__.py` without the copyright info (which we should expect), the file will get overwritten.
2. The official document https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages stated the following. I try to follow the doc rather instead of relying on the actual implementation. 

`The __init__.py file for the namespace package needs to contain only the following:`

```
__import__('pkg_resources').declare_namespace(__name__)
```